### ISSUE TITLE
Sync date in events page

### ISSUE THREAD

**Issue by Nutster2454** [smurphboy/OARCLogger#10]
Sync date so with a button it matches date at the start date , when one day events.

![image](https://user-images.githubusercontent.com/24573564/226136684-c11a8ee0-cba8-4b2f-b62f-d5ac8a5dedcf.png)

**Comment by smurphboy** [comment]
Yep - easy enough to add a button like we have for QSO on / QSO off date on the QSO form
